pkg/tarmak/vault/vault.go

```diff
@@ -109,20 +110,25 @@ func (v *Vault) TunnelFromFQDNs(vaultInternalFQDNs []string, vaultCA string) (in
 
 			if health.Standby == false && health.Sealed == false {
```

Comment:
Maybe we have to check here for health.Initialized

Reply:
agreed